### PR TITLE
Improve btrfs version check

### PR DESCRIPTION
**Pull request by phacops**
We'll probably drop the btrfs code anytime soon but in the meantime, some small improvements:
- Allow to have a newer major version than the one specified
- Fix an error message typo where the needed minor version was shown twice
- Split the error message in a more accurate way (my version was reporting `btrfs-tools v4.1.2` instead of `Btrfs v4.1.2` like we were waiting for)

### CODE REVIEW

File: src/btrfs/btrfs.go [hunk 1]
```diff
@@ -655,29 +655,32 @@ func checkVersion() {
...
-	split := strings.Split(version, ".")
-	if len(split) != 2 {
+	splittedVersion := strings.Split(versionString, " v")
+	if len(splittedVersion) != 2 {
 		checkVersionErr = fmt.Errorf("unknown version string: %s", versionString)
 		return
 	}
+	split := strings.Split(splittedVersion[1], ".")
+	if len(split) >= 2 {
+	}
```
Comment (peter-edge) [inline src/btrfs/btrfs.go]: Blank if?

**Comment by peter-edge** [comment]
LGTM++, just one comment to fix

File: src/btrfs/btrfs.go [hunk 2]
```diff
@@ -655,29 +655,34 @@ func checkVersion() {
...
 	}
-	major, err := strconv.ParseInt(split[0], 10, 64)
-	if err != nil {
-		checkVersionErr = err
-		return
-	}
-	if major < majorVersion {
-		checkVersionErr = fmt.Errorf("need at least btrfs version %d.%d, got %s", majorVersion, minorVersion, version)
+	split := strings.Split(splittedVersion[1], ".")
+	if len(split) < 2 {
```
Comment (peter-edge) [inline src/btrfs/btrfs.go]: I think it should be `if len(split) == 2 {`

Comment (peter-edge) [inline src/btrfs/btrfs.go]: !=

Comment (phacops) [inline src/btrfs/btrfs.go]: A version can be `4.1.2` for example. We're splitting on `.` so we would have a `len(split)` of 3 if I'm correct. So the only case where we want to return an error is when they're less than a major and minor version, no?

Comment (peter-edge) [inline src/btrfs/btrfs.go]: ah ok, then check if != 2 or != 3

**Comment by peter-edge** [comment]
LGTM

**Pull request merged**